test/resources/microshift-process.resource

```diff
@@ -86,6 +82,8 @@ Restart MicroShift
     Systemctl With Retry    restart    microshift.service
     Wait Until MicroShift Process ID Changes    ${ushift_pid}
     Wait Until MicroShift Etcd Process ID Changes    ${ushift_etcd_pid}
+    # Kubeconfig may change between restarts, ensure we are taking the latest one.
+    Setup Kubeconfig
```

Comment:
This is interesting, is this because of IP change that we want to fetch new kubeconfig?
I'm not sure if that should be a rule for every test.

Changing IP in the kubeconfig is one thing, but kubeconfig getting changed completely is another.

Would you consider this a bug? I would be pretty annoyed if my kubeconfig stopped working all of the sudden on some remote device

Reply:
This is related to the IP change, yes. A simple restart in between different IPs and calling this keyword may yield an unusable kubeconfig because the `server` part changed.
If the CAs would get rotated (or client certs) the same thing could happen. I would not say its a bug, its like a safety net. Setting up kubeconfig at the beginning does not cover everything.